modnet/tests/test_benchmark.py

```diff
@@ -30,11 +30,50 @@ def test_train_small_model_benchmark(subset_moddata, tf_session):
         "errors",
         "scores",
         "best_presets",
+        "model",
     )
 
     assert all(key in results for key in expected_keys)
     assert all(len(results[key]) == 5 for key in expected_keys)
 
+@pytest.mark.skip(msg="Slow test")
```

Comment:
We already have a "mark" for slow tests (see https://github.com/ppdebreuck/modnet/blob/master/pytest.ini), which skips them in the CI:

```suggestion
@pytest.mark.slow
```